virtcontainers/qemu.go

```diff
@@ -1307,16 +1307,18 @@ func (q *qemu) hotplugAddCPUs(amount uint32) (uint32, error) {
 		driver := hc.Type
 		cpuID := fmt.Sprintf("cpu-%d", len(q.state.HotpluggedVCPUs))
 		socketID := fmt.Sprintf("%d", hc.Properties.Socket)
+		dieID = fmt.Sprintf("%d", hc.Properties.Die)
 		coreID := fmt.Sprintf("%d", hc.Properties.Core)
 		threadID := fmt.Sprintf("%d", hc.Properties.Thread)
 
 		// If CPU type is IBM pSeries or Z, we do not set socketID and threadID
 		if machine.Type == "pseries" || machine.Type == "s390-ccw-virtio" {
 			socketID = ""
 			threadID = ""
+			dieID = ""
```

Comment:
Nit: the comment needs updating for dieID :) I'd be tempted to change it to:

```go
// If CPU type is IBM pSeries or Z, we do not set the following.
```

/cc @alicefr since this affects s390x.

Reply:
I think she agrees with this change https://github.com/intel/govmm/pull/103#issue-299840547